docs/fides/docs/quickstart/docker.md

```diff
@@ -18,7 +39,7 @@ This is a reference file that you can copy/paste into a local `docker-compose.ym
 ```docker-compose title="docker-compose.yml"
 services:
   fidesctl:
-    image: ethyca/fidesctl:1.0.0
+    image: ethyca/fidesctl:1.2.0
```

Comment:
other than `latest` im not really sure how we'd do it. Just having the whole compose in here to begin with feels like a liability but i'm going to leave it for now since we encourage people to use docker (that can change though, we've made other methods much easier)